roles/lib_openshift/src/test/integration/oc_process.yml

```diff
@@ -66,13 +64,14 @@
       reconcile: false
     register: procout
 
+  # These assertions could fail due to changes in item order
   - assert:
       that:
       - not procout.changed
-      - procout.results.results['items'][0]['metadata']['name'] == 'testdb'
-      - procout.results.results['items'][0]['kind'] == 'Service'
-      - procout.results.results['items'][1]['metadata']['name'] == 'testdb'
-      - procout.results.results['items'][1]['kind'] == 'DeploymentConfig'
+      - procout.module_results.results['items'][0]['metadata']['name'] == 'testdb'
+      - procout.module_results.results['items'][0]['kind'] == 'Secret'
+      - procout.module_results.results['items'][2]['metadata']['name'] == 'testdb'
```

Comment:
Item 1 is being skipped?

Reply:
Same as above, but DeploymentConfig is now the third item.  More can be done to make these integration tests work better, I was just making sure that the data returned was correct and what would verify the module worked.